wdl-format/src/config/builder.rs

```diff
@@ -30,37 +39,62 @@ pub type Result<T> = std::result::Result<T, Error>;
 pub struct Builder {
     /// The number of characters to indent.
     indent: Option<Indent>,
+    /// The maximum line length.
+    max_line_length: Option<MaxLineLength>,
 }
 
 impl Builder {
     /// Creates a new builder with default values.
-    pub fn new() -> Self {
-        Default::default()
+    pub fn new(indent: Option<Indent>, max_line_length: Option<MaxLineLength>) -> Result<Self> {
```

Comment:
I'm not use to seeing parameters to a builder constructor.

Perhaps we could leave this as `new()` and perform the `unwrap_or_default` logic in `try_build`?

It also seems like we have good defaults for both `ident` and `max_line_length`, so I don't see how `try_build` would need to be fallible (so simply change to `build`?).